Spigot-API-Patches/0256-Make-ProjectileHitEvent-cancellable.patch

```diff
@@ -0,0 +1,42 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: jmp <jasonpenilla2@me.com>
+Date: Tue, 29 Dec 2020 15:05:50 -0800
+Subject: [PATCH] Make ProjectileHitEvent cancellable
+
+
+diff --git a/src/main/java/org/bukkit/event/entity/ProjectileHitEvent.java b/src/main/java/org/bukkit/event/entity/ProjectileHitEvent.java
+index 809c1098c16fe52eda47ae2335afab30e36746b2..4a734be2ca03711edf7c885242550701d8e5771c 100644
+--- a/src/main/java/org/bukkit/event/entity/ProjectileHitEvent.java
++++ b/src/main/java/org/bukkit/event/entity/ProjectileHitEvent.java
+@@ -10,12 +10,14 @@ import org.jetbrains.annotations.Nullable;
+ 
+ /**
+  * Called when a projectile hits an object
++ * <p> This Event is made cancellable by Paper
```

Comment:
I dont really feel this comment is relevant. you can see that its callable since it implements the interface, what spigot does is not relevant.